api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -780,8 +780,8 @@
     public static final String EXITCODE = "exitcode";
     public static final String TARGET_ID = "targetid";
     public static final String FILES = "files";
-    public static final String SRC_POOL = "srcpool";
-    public static final String DEST_POOLS = "destpools";
+    public static final String SRC_POOL = "sourcepool";
```

Comment:
Are these in use by any API as it could break API backward compatibility (request/response)?

Reply:
no, they are new in this PR